### PR TITLE
Heightmap

### PR DESCRIPTION
**Pull request by noor-gate**

### CODE REVIEW

File: api/flaskr/files.py [hunk 1]
```diff
@@ -56,17 +57,32 @@ def upload_terrain_data() -> Response:
...
     valid_uploads = list(filter(is_valid_upload, uploaded_files))
-    if not valid_uploads:
+    if not valid_uploads or len(valid_uploads) != 2:
         print("No valid uploads")
         return Response('invalid image(s)', 400)
 
-    for upload in valid_uploads:
+    paths = []
+    pcdname = ""
+    for i, upload in enumerate(valid_uploads):
```
Comment (woocashh) [inline api/flaskr/files.py:66]: Why do you loop but then just use one file?

File: api/pcd_generator/pointcloud.py [hunk 2]
```diff
@@ -1,58 +1,59 @@
...
+    # image_location = pcd_generator_dir + '/thing.tif'
+    # depth_map_location = pcd_generator_dir + '/thing2.tif'
+
+    # Load in the images
+    depth_map = mpimg.imread(depth_map_location)
+    image = mpimg.imread(image_location)
+
+    rows, cols, _ = depth_map.shape
+
+    # Cut down pixels for time purpose
```
Comment (woocashh) [inline api/pcd_generator/pointcloud.py:17]: Should this not be relative to image size not absolute?

**Review by woocashh — COMMENTED** [review]
Just a couple of comments regarding logic of code

**Review by andreigramescu — APPROVED** [review]

File: api/flaskr/files.py (hunk 1 above)
Comment (TheAdamG) [reply api/flaskr/files.py:66]: We expect there to be two files, we loop through both of them but if any of theme have an invalid name, we return

**Pull request merged**